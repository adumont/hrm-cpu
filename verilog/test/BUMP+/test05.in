39 c6
11 2a
40 32
57 68
17 79
50 1f
18 d9
22 99
30 d7
16 46
24 af
10 20
35 fc
49 ed
38 34
43 42
31 23
23 ce
01 bf
26 54
02 26
09 ad
23 b7
08 62
11 e2
12 e1
59 57
57 64
45 2e
03 97
59 bc
31 1d
11 ee
43 ed
