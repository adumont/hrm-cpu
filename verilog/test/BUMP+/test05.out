c7
2b
33
69
7a
20
da
9a
d8
47
b0
21
fd
ee
35
43
24
cf
c0
55
27
ae
b8
63
e3
e2
58
65
2f
98
bd
1e
ef
ee
